<launch>
  <node name="compare22" pkg="grad_traj_optimization" type="compare22" output="screen">
  <!-- path searching -->
    <param name="search/max_tau" value="1.0" type="double"/>
    <param name="search/init_max_tau" value="0.8" type="double"/>
    <param name="search/max_vel" value="3.0" type="double"/>
    <param name="search/max_acc" value="2.0" type="double"/>
    <param name="search/w_time" value="10.0" type="double"/>
    <param name="search/horizon" value="50.0" type="double"/>
    <param name="search/lambda_heu" value="1.0" type="double"/>
    
    <param name="search/resolution_astar" value="0.75" type="double"/>

    <param name="search/time_resolution" value="0.8" type="double"/>
    <param name="search/margin" value="0.1" type="double"/>

    <param name="search/allocate_num" value="100000" type="int"/>
    <param name="search/check_num" value="5" type="int"/>

  <!-- sdf_map -->
    <param name="sdf_map/origin_x" value="-20.0" type="double"/>
    <param name="sdf_map/origin_y" value="-20.0" type="double"/>
    <param name="sdf_map/origin_z" value="-2.0" type="double"/>

    <param name="sdf_map/map_size_x" value="40.0" type="double"/>
    <param name="sdf_map/map_size_y" value="40.0" type="double"/>
    <param name="sdf_map/map_size_z" value="5.0" type="double"/>

    <param name="sdf_map/resolution_sdf" value="0.2" type="double"/>
    <param name="sdf_map/ceil_height" value="-1.0" type="double"/>
    <param name="sdf_map/update_rate" value="10.0" type="double"/>
    <param name="sdf_map/update_range" value="5.0" type="double"/>
    <param name="sdf_map/inflate" value="0.1" type="double"/>
    <param name="sdf_map/radius_ignore" value="1.0" type="double"/>
  </node>
</launch>
 
